internal/services/contracts_events_consumer.go

```diff
@@ -126,6 +128,10 @@ func (c *ContractsEventsConsumer) Process(ctx context.Context, event *shared.Clo
 			return c.handleVehicleTransferEvent(ctx, &data)
 		case PrivilegeSet:
 			return c.handlePrivilegeSetEvent(ctx, &data)
+		case TokensTransferredForDevice:
+			return c.handleTokensTransferredForDevice(ctx, &data)
+		case TokensTransferredForConnectionStreak:
+			return c.handleTokensTransferredForConnectionStreak(ctx, &data)
```

Comment:
Actually, this is suspicious—shouldn't these be coming from the rewards contract? I think we need to add a setting.